pkg/prowloader/prow.go

```diff
@@ -181,6 +181,7 @@ func fetchJobsJSON(prowURL string) ([]byte, error) {
 	if err != nil {
 		return nil, err
 	}
+	defer resp.Body.Close()
```

Comment:
I think this is needed / best practice but if not let me know

Reply:
i saw [this](https://golangbyexample.com/resposne-body-closed-golang) and I agree